src/components/activity/FormSetDates.js

```diff
@@ -0,0 +1,64 @@
+import {useState} from 'react';
+
+function FormSetDates({addActivity}) {
+    const [activity, setActivity] = useState({
+        activity: "",
+        startTime: "00:00",
+        endTime: "00:00",
+      });
+
+    const handleChange = (event) => {
+        setActivity({ ...activity, [event.target.name]: event.target.value });
+      };
+    
+    const handleSubmit = (event) => {
+    event.preventDefault();
+    addActivity(activity);
+    setActivity({  activity: "", startTime: "00:00", endTime: "00:00", });
```

Comment:
You could define a constant object, e.g. `initialActivityState` outside of the component function and use this to initialize the `activity` state and to reset your state in `handleSubmit`. If you want to change your initial object later, you don't need to do this at two places. (DRY principle)